TheMET/TheMET/ViewControllers/CatalogViewController.swift

```diff
@@ -14,6 +14,10 @@ class CatalogViewController: UIViewController {
     
     private let aboutButton: UIButton = UIButton()
     
+    private let scrollView: UIScrollView = UIScrollView()
+    
+    private let scrollContainer: UIView = UIView()
```

Comment:
You don't need a `scrollContainer`, use constraints from the `scrollView.contentLayoutGuide` instead